<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dodajciagnik</class>
 <widget class="QDialog" name="dodajciagnik">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QDateEdit" name="UbezpieczenieEdit">
     <property name="displayFormat">
      <string>yyyy-MM-dd</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="ok">
     <property name="text">
      <string>ok</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Ubezpieczenie</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDateEdit" name="Opony1Edit">
     <property name="displayFormat">
      <string>yyyy-MM-dd</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLineEdit" name="nrRejEdit">
     <property name="placeholderText">
      <string>Wpisz numer Rejestracyjny Ciagnika</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QDateEdit" name="CzytanieTachoEdit">
     <property name="displayFormat">
      <string>yyyy-MM-dd</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>PrzeglÄ…d</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QDateEdit" name="Opony2Edit">
     <property name="displayFormat">
      <string>yyyy-MM-dd</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Legalizacja Tachografu</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QDateEdit" name="LegalizacjaTachoEdit">
     <property name="displayFormat">
      <string>yyyy-MM-dd</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QDateEdit" name="przegladEdit">
     <property name="displayFormat">
      <string>yyyy-MM-dd</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Czytanie Tachografu</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Opony 1</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Opony 2</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
